Minor tweaks to support Loom X,Y coordinates with AnnDataWrapper

keller-mark opened the pull request:
Fixes #24

ilan-gold commented:
Have you experimented with https://anndata.readthedocs.io/en/latest/anndata.read_loom.html?

keller-mark commented:
No, I had not seen that, I will try it

keller-mark commented:
I have removed the `LoomWrapper` class and just updated the `create_cells_json` of `AnnDataWrapper` to support spatial coordinates. AnnData's `read_loom` works great
